README.md

```diff
@@ -60,8 +60,18 @@ running it.
 
 ### Compiling from source:
 
-You can also build from source by cloning this project and running
-`git submodule update --init --recursive; make install` (Xcode 10.2 or later).
+You can also build from source by cloning this project and following the steps according to the Xcode version (10.2 or later)
+
+#### Xcode 10.2.x or 10.3.x
+
+- Run `git submodule update --init --recursive; make install`
```

Comment:
what about the other code generated stuff that `make install` was generating? 🤔 

Things like: https://github.com/realm/SwiftLint/blob/master/Makefile#L41

Reply:
thank you @crsantos, great point! I'll make sure to double-check that once the issue with SourceKitten is resolved, I didn't have any problems without `make install` on Xcode 11 except for SourceKitten but it may be that I would start to have some after SourceKitten succeeds 😅

Reply:
the build succeeded without `make install` when using the branch from https://github.com/realm/SwiftLint/pull/2894 but I suppose `make install` is for actually installing SwiftLint to the machine. I should definitely add it back then, but I'm curious (sorry I'm not very acquainted to SPM), why does it fetch, clone and resolve all of the dependencies again if I have already built the project successfully from inside Xcode 11? It felt to me like the script was doing redundant work that SPM already does when loading the project. If this is all my misunderstanding, I apologize and close this PR 👍